lms/djangoapps/instructor/views/tools.py

```diff
@@ -280,7 +280,10 @@ def generate_course_forums_d3(url_handle):
     previous_date = -1
     previous = None
     rows = [['Date', 'New Threads', 'Responses', 'Comments']]
-    for date, comment_type, number, up_vote, down, net in reader: # pylint: disable=unused-variable
+    print 'starting'
+    for row in reader:
```

Comment:
I see. It looks previously the query returned date, comment_type, up_vote, down, and net. Of these up_vote, down, and net were unused. The new query returns date, comment_type, number new, and votes. Of these, you only need the first 3. Since number_new and up_vote correspond to the same field, the code is backward compatible